web_src/js/features/formatting.js

```diff
@@ -11,4 +11,14 @@ export function initFormattingReplacements() {
       el.textContent = formatted;
     }
   }
+
+  // for each <time></time> tag, if it has the data-date-format, format
+  // the text according to the user's chosen locale
+  const locale = document.getElementsByTagName('html')[0].lang;
+  const formatter = new Intl.DateTimeFormat(locale, {year: 'numeric', month: 'long', day: 'numeric'});
+  for (const timeElement of document.getElementsByTagName('time')) {
```

Comment:
You can just `document.querySelectorAll('time[data-format]')` and skip the `if` condition below.